draft-ietf-sframe-enc.md

```diff
@@ -179,42 +184,47 @@ complexity, at the cost of higher bandwidth consumption.  Some quantitative
 discussion of these trade-offs is provided in {{overhead-analysis}}.
 
 As noted above, however, SFrame is a general media encapsulation, and can be
-applied in other scenarios.  The precise efficiency and complexity trade-offs
-will depend on the environment in which SFrame is being integrated.
+applied in other scenarios.  The important thing is that the sender and
+receivers of an SFrame-encrypted object agree on that object's semantics.
+SFrame does not provide this agreement; it must be arranged by the application.
 
 ~~~ aasvg
       +--------------------------------------------------------+
       |                                                        |
       |  +----------+      +-------------+      +-----------+  |
  .-.  |  |          |      |             |      |   SRTP    |  |
```

Comment:
Should we make this a bit more general than SRTP? Could this say HBH instead? Same for the Unprotect on line 220